src/main/java/org/broadinstitute/hellbender/tools/walkers/qc/CheckPileup.java

```diff
@@ -64,6 +66,10 @@
     @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME, shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME, doc = "Output file (if not provided, defaults to STDOUT).", optional = true)
     public File outFile = null;
 
+    // This is to check for previous versions of samtools without having overlap detection
+    @Argument(fullName = "ignore_overlaps", shortName = "x", doc = "Disable read-pair overlap detection.", optional = true)
```

Comment:
I used the sames as in samtools mpileup, but it is true that it is not a good short name. I changed it.